<ui version="4.0" >
 <class>QPalringoSigninWindow</class>
 <widget class="QDialog" name="QPalringoSigninWindow" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sign In</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../resources/qpalringo.qrc" >
    <normaloff>:/misc/sysTrayIcon.png</normaloff>:/misc/sysTrayIcon.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Palringo Account Details</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="EmailLabel" >
        <property name="text" >
         <string>&amp;Email address</string>
        </property>
        <property name="buddy" >
         <cstring>EmailEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EmailEdit" />
      </item>
      <item>
       <widget class="QLabel" name="PasswordLabel" >
        <property name="text" >
         <string>&amp;Password</string>
        </property>
        <property name="buddy" >
         <cstring>PasswordEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordEdit" >
        <property name="inputMask" >
         <string/>
        </property>
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RememberMe" >
     <property name="text" >
      <string>&amp;Remember me</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="AutoSignin" >
     <property name="text" >
      <string>&amp;Sign me in automatically</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="OkButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>EmailEdit</tabstop>
  <tabstop>PasswordEdit</tabstop>
  <tabstop>RememberMe</tabstop>
  <tabstop>AutoSignin</tabstop>
  <tabstop>OkButton</tabstop>
  <tabstop>CancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/qpalringo.qrc" />
 </resources>
 <connections/>
</ui>
